codespell_lib/_codespell.py

```diff
@@ -631,9 +631,11 @@ def main(*args):
                     del dirs[:]
                     continue
                 for file_ in files:
-                    if glob_match.match(file_):
+                    if glob_match.match(file_):  # skip files
```

Comment:
I've left this line in for safety and backwards compatibility, but I think it could be dropped in theory.

Reply:
Agreed it's probably not necessary but I'm no `glob` expert so fine leaving it in